docs/Set-PSReadLineOption.md

```diff
@@ -552,6 +555,35 @@ Accept pipeline input: false
 Accept wildcard characters: False
 ```
 
+### -ViModeChangeHandler
+
+When the `ViModeIndicator` is set to `Script`, the script block provided will be invoked every time the mode changes. The script block is provided one argument of type `ViMode`. The following example changes the cursor to a blinking block in command mode and a blinking line in insert mode.
+
+```powershell
+{
+    if ($args[0] -eq 'Command')
+    {
+        Write-Host "`e[1 q"
+    }
+    else
+    {
+        Write-Host "`e[5 q"
+    }
+}
+```
```

Comment:
Hm, glad platyPS was able to handle that 😅 
Maybe it makes more sense to put that into Example section and reference from here?